etc/docker/enterprise-gateway/start-enterprise-gateway.sh

```diff
@@ -26,15 +26,15 @@ export EG_LOG_LEVEL=${EG_LOG_LEVEL:-DEBUG}
 export EG_CULL_IDLE_TIMEOUT=${EG_CULL_IDLE_TIMEOUT:-43200}  # default to 12 hours
 export EG_CULL_INTERVAL=${EG_CULL_INTERVAL:-60}
 export EG_CULL_CONNECTED=${EG_CULL_CONNECTED:-False}
-export EG_KERNEL_WHITELIST=${EG_KERNEL_WHITELIST:-"['r_docker','python_docker','python_tf_docker','scala_docker','spark_r_docker','spark_python_docker','spark_scala_docker']"}
+export EG_KERNEL_WHITELIST=${EG_KERNEL_WHITELIST:-"'r_docker','python_docker','python_tf_docker','scala_docker','spark_r_docker','spark_python_docker','spark_scala_docker'"}
 export EG_DEFAULT_KERNEL_NAME=${EG_DEFAULT_KERNEL_NAME:-python_docker}
```

Comment:
Because the helm charts produce a bracketed value, and this may be useful for b/c purposes anyway, I found I needed something like the following to 'scrub' the brackets...
```bash

# Remove potential brackets from kernel whitelist
export EG_KERNEL_WHITELIST=`echo ${EG_KERNEL_WHITELIST} | sed 's/\[//' | sed 's/\]//'`
```

Reply:
A single `sed` statement can also be used:
```bash
export EG_KERNEL_WHITELIST=`echo ${EG_KERNEL_WHITELIST} | sed 's/[][]//g'`
```

Reply:
Neat idea! I should look at `sed` more often when working with shell scripts.

Reply:
I've updated the PR. I thought a multi-line approach would be nicer. And a single export more logical? Let me know if it needs any changes.

Reply:
This looks great.  Would you mind adding a comment prior to the `sed` statement, something to the effect of: "Strip off any surrounding brackets that may have been set already".  That particular `sed` statement isn't very obvious, especially the way the brackets are organized within it - which matters (eg, `'s/[[]]//g'` doesn't work, although more intuitive).

Reply:
Good idea! The `sed` statement does requiring some expert parsing to become obvious.